src/features/course-main/course-main.tsx

```diff
@@ -10,6 +10,9 @@ interface CourseMainProps {
   type?: CourseType;
 }
 
+const MS_IN_2_WEEKS = 1209600000;
```

Comment:
If we wouldn't like to use work-with-date library I suggest to create some function which takes `period-type` (month, day, week) and `periods-value` and returns value in ms

Reply:
Added date library Day.js